services/ui-src/src/types/index.ts

```diff
@@ -331,6 +332,11 @@ export interface InputChangeEvent extends React.ChangeEvent<HTMLInputElement> {}
 
 export type { IconType } from "react-icons";
 
+export interface AccessMeasureAssociatedPlan {
+  name: string;
+  value: string;
+}
+
```

Comment:
Note: We probably won't need this after the (likely) design changes to the card bottom.